lib/cocoapods/target/pod_target.rb

```diff
@@ -688,5 +716,43 @@ def uses_modular_headers?(only_if_defines_modules = true)
     def create_build_settings
       BuildSettings::PodTargetSettings.new(self)
     end
+
+    # Computes the destination sub-directory in the sandbox
+    #
+    # @param  [Pathname] headers_sandbox
+    #         The sandbox where the header links should be stored for this
+    #         Pod.
+    #
+    # @param  [Sandbox::FileAccessor] file_accessor
+    #         The consumer file accessor for which the headers need to be
+    #         linked.
+    #
+    # @param  [Array<Pathname>] headers
+    #         The absolute paths of the headers which need to be mapped.
+    #
+    # @return [Hash{Pathname => Array<Pathname>}] A hash containing the
+    #         headers folders as the keys and the absolute paths of the
+    #         header files as the values.
+    #
+    def header_mappings(headers_sandbox, file_accessor, headers)
```

Comment:
if this method lives in `pod_target` I dont think you need `headers_sandbox` as a parameter?

Reply:
Good point